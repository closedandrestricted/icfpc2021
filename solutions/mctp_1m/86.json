{"vertices":[[53,57],[56,64],[47,59],[41,64],[49,58],[55,63],[55,71],[51,65],[47,62],[42,52],[48,50],[57,56],[54,63],[50,58],[43,55],[45,57],[53,44],[58,49],[50,58],[46,52],[51,58],[50,51],[58,56],[54,51],[48,59],[41,56],[44,55],[54,45],[59,49],[46,53],[52,52],[37,48],[50,53],[47,47],[60,56],[53,55],[46,49],[42,43],[44,53],[52,51],[58,49],[49,49],[46,42],[41,47],[45,53],[55,43],[48,41],[46,36],[41,30],[44,45],[51,48],[44,36],[49,30],[34,33],[48,42],[52,36],[43,31],[40,38]]}