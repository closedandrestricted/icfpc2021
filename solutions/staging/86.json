{"vertices":[[35,43],[29,48],[32,66],[26,71],[35,32],[40,38],[30,56],[35,61],[22,78],[30,21],[29,28],[36,44],[30,49],[31,67],[25,71],[37,19],[34,33],[39,39],[36,54],[34,61],[43,23],[39,29],[42,45],[43,52],[41,62],[41,69],[51,22],[47,33],[48,40],[50,57],[49,64],[49,73],[55,28],[54,35],[53,44],[55,51],[56,66],[49,65],[57,21],[60,40],[59,47],[59,57],[61,63],[63,25],[60,32],[65,50],[59,49],[68,63],[74,68],[63,39],[61,45],[57,56],[64,57],[73,75],[64,50],[57,49],[70,60],[73,67]]}