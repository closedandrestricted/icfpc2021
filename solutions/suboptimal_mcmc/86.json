{"vertices":[[22,54],[21,61],[32,75],[29,82],[34,53],[28,50],[29,62],[34,68],[37,81],[39,39],[38,46],[34,52],[36,59],[41,67],[41,74],[33,35],[39,39],[36,46],[43,57],[47,63],[27,31],[32,37],[41,42],[44,49],[53,59],[59,64],[21,35],[31,30],[37,35],[52,46],[57,52],[67,60],[17,29],[24,26],[44,33],[49,39],[61,46],[66,52],[15,22],[32,25],[39,28],[56,36],[63,39],[19,16],[25,21],[45,25],[50,31],[63,32],[66,25],[33,20],[40,20],[56,26],[62,25],[72,20],[45,16],[50,22],[64,18],[70,13]]}