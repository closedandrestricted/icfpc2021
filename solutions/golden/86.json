{"vertices":[[51,22],[46,16],[29,21],[22,23],[62,12],[57,18],[40,21],[34,26],[20,30],[72,19],[65,19],[53,24],[46,26],[35,33],[28,30],[68,25],[61,26],[55,31],[44,33],[39,39],[65,31],[58,33],[51,37],[45,41],[35,44],[37,50],[63,39],[57,41],[51,45],[42,49],[35,52],[30,54],[65,46],[58,49],[49,52],[43,57],[29,56],[22,54],[69,52],[57,56],[51,59],[37,60],[30,63],[68,59],[61,62],[49,66],[42,66],[28,70],[21,72],[54,66],[51,73],[42,74],[35,72],[18,79],[55,79],[48,79],[30,77],[25,83]]}